src/Xamarin.Android.Build.Tasks/Tests/Xamarin.Android.Build.Tests/Xamarin.Android.Build.Tests.Shared.projitems

```diff
@@ -30,4 +31,4 @@
     <Compile Include="$(MSBuildThisFileDirectory)Utilities\MockBuildEngine.cs" />
     <Compile Include="$(MSBuildThisFileDirectory)Utilities\MonoAndroidHelperTests.cs" />
   </ItemGroup>
-</Project>
+</Project>
```

Comment:
I think because of unix we like the line ending here. So you can do `cat foo.projitems` it won't mess up your terminal.

Reply:
no idea, this is VS doing whatever it does. should I care? 🤦‍♂️ 

If the IDE is doing this, I don't think I can stop it from doing it

Reply:
You can edit the file and add the line ending back, though. 

Just because "my IDE did it", doesn't mean we can't fix it. 😄

Reply:
Then there's a misconfiguration on the `.gitattributes`, I'd say. Why would you care for `cat`ing .projitems? Again, if contributors are intentionally supposed to be manually tweaking line endings in a file that is entirely managed by the IDE (since the .projitems is a *generated* file from VS by the `.shproj` shared project), then something is quite not right.

Reply:
I think you just need to press "return" at the end of the file. I'll add a commit to fix any formatting, no problem.